runtime/standalone/src/system_config.rs

```diff
@@ -185,3 +188,241 @@ impl pallet_utility::Config for Runtime {
     type PalletsOrigin = OriginCaller;
     type WeightInfo = pallet_utility::weights::SubstrateWeight<Runtime>;
 }
+pub struct BaseCallFilter;
+impl Contains<Call> for BaseCallFilter {
+    fn contains(c: &Call) -> bool {
+        match c {
+            // System support
+            Call::System(_) => true,
+            // Call::ParachainSystem(_) => true,
+            Call::Timestamp(_) => true,
+            // Call::Preimage(_) => true,
+            // Call::Scheduler(_) => true,
```

Comment:
Just remove anything to do with parachains here

Reply:
Done